contentcuration/contentcuration/utils/format.py

```diff
@@ -6,12 +8,12 @@ def format_size(value):
     TB = float(KB ** 4) # 1,099,511,627,776
 
     if B < KB:
-        return '{0}'.format(B), 'B'
+        return '{0}'.format(B), _('B')
```

Comment:
Do the numbers/units then get combined in an RTL sensitive way? Also, do we format numerals by locale?

Reply:
That's a good point. I'll make a change here